cmd/posix.go

```diff
@@ -209,6 +209,16 @@ func isDirEmpty(dirname string) bool {
 	return true
 }
 
+func testFSCapabilities(path string) error {
```

Comment:
You should add this into getValidPath() which checks for permissions @vadmeste

Reply:
On a related note, could we move this check for DIRECTIO support to `pkg/disk` where other DIRECTIO functions are defined?